doc/howto/network_bridge_firewalld.md

```diff
@@ -1,52 +1,109 @@
-(network-bridge-firewalld)=
-# How to configure Firewalld
+---
+discourse: 10034,9953
```

Comment:
This adds a link to the Docker forum post (and the nftables one), but should we have a section about Docker in here as well?

Reply:
I added a short section now.

Reply:
@stgraber are you happy covering docker in our docs?